nss-day-cohort-28/Class-Website
# Update Cohort Information

## Changes Made

1. Added all images, bios, links, etc to cohort.json. Site now displays all info for all classmates

1. Added 'hired' status in json file that will display a badge as classmates accept job offers

## Steps To Test

1. `git pull origin master`
1.  `git fetch --all`
1.  `git checkout update-images`
1.  start http server
1.  start json server 1: data/cohort.json
1. start json server 2: data/techs.json
1. Check all links, modals and info
